### PR TITLE
fix: should throw when unexpect assert type

### PR DESCRIPTION
**Pull request by atian25**
backport to 3.x due to https://github.com/eggjs/egg-cluster/pull/78#discussion_r221143298

### CODE REVIEW

**Comment by atian25** [comment]
怪了，单元测试没触发

File: lib/coffee.js [hunk 1]
```diff
@@ -120,25 +120,25 @@ Coffee.prototype.debug = function(level) {
...
 
 Coffee.prototype.expect = function(type, value) {
-  if (acceptType.indexOf(type) > -1) {
-    var rule = new Rule(value);
-    if (this.complete) {
-      assertion([ rule ], this[type], 'match ' + type);
-    } else {
-      this.waitAssert[type].push(rule);
-    }
+  if (acceptType.indexOf(type) === -1) throw new Error(`unexpect assert type: ${type}`);
```
Comment (popomore) [inline lib/coffee.js]: 加一个 should be acceptTyp？

Comment (atian25) [reply lib/coffee.js]: done

> unexpect assert type: `unacceptkey`, only accept: [stdout,stderr,code,error]

**Comment by atian25** [comment]
3.3.3

**Pull request merged**